db/migrate/20170227013245_create_order_items.rb

```diff
@@ -0,0 +1,11 @@
+class CreateOrderItems < ActiveRecord::Migration
+  def change
+    create_table :order_items do |t|
+      t.references :item, index: true, foreign_key: true
+      t.references :order, index: true, foreign_key: true
+      t.integer :quantity
+      t.string :status
```

Comment:
Good call on the status being on the joins table. Being able to have a status for each line item makes this way more dynamic in terms of order management.